tests/framework/installer/ceph_installer.go

```diff
@@ -43,7 +43,7 @@ import (
 
 const (
 	// test with the latest nautilus build
-	nautilusTestImage = "ceph/ceph:v14.2.12"
```

Comment:
The upgrade test looks like it's failing because Rook v1.5.x won't work with partitions, so I believe the upgrade test will still need to use v14.2.12.

Reply:
Hum ok so there is nothing to do at the end, right?

Reply:
The multi-cluster test also uses Nautilus. So the multi-cluster test could use v14 and the upgrade test would still use v14.2.12.